.github/workflows/test-db-changes.yml

```diff
@@ -82,3 +82,13 @@ jobs:
           fi
           index=$((index+1))
         done
+
+    - name: Prevent hosts foreign keys
+      run: |
+        MATCHES=$(git diff --name-only origin/main | grep "migrations/" | xargs grep -l "REFERENCES hosts(id)")
```

Comment:
Should probably be less strict on the `REFERENCES` match, i.e. case-insensitive and allowing arbitrary spacing for the various coding preferences (I think something like `grep -i -E 'references\s*hosts\s*\(\s*id\s*\)' ` would work).

Reply:
nice!  thanks for the regex,  i'll test it out!

Reply:
dusted off this PR, and that regex works great.  Last commit removes my test migration.